lib/modules/apostrophe-events-calendar-widgets/lib/routes.js

```diff
@@ -0,0 +1,15 @@
+var _ = require('lodash');
+var moment = require('moment');
+
+module.exports = function (self, options) {
+  self.addRoutes = function () {
+    self.route('post', 'calendar', async function(req, res) {
+      try {
```

Comment:
Mixing modern try/catch with `var`. It feels inconsistent and would benefit from having the apos eslint on this.

Reply:
so I made the new module all const/let/async .. I added our eslint config and fixed this module but did not cleanup the entire bundle .. was just trying to scope this branch A LITTLE BIT